packages/express/mixin.core.js

```diff
@@ -8,37 +8,20 @@ const { Mixin } = require('@untool/core');
 const uri = require('./lib/uri');
 
 class ExpressMixin extends Mixin {
-  create(method) {
-    const create = require(method === 'static'
-      ? './lib/static'
-      : './lib/serve');
+  createServer(mode) {
+    const create = require(mode === 'static' ? './lib/static' : './lib/serve');
```

Comment:
How about writing this like: `const create = mode === 'static' ? require('./lib/static') : require('./lib/serve');`?